.github/workflows/upload.yml

```diff
@@ -0,0 +1,28 @@
+name: Upload lte-core-interface-lib
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  charmhub-upload:
+    if: github.ref_name == 'main'
```

Comment:
You don't need this `if`, because this workflow runs only on push to main anyway.

Reply:
Removed.